agent/job_runner.go

```diff
@@ -248,7 +249,9 @@ func NewJobRunner(l logger.Logger, scope *metrics.Scope, ag *api.AgentRegisterRe
 func (r *JobRunner) Run() error {
 	r.logger.Info("Starting job %s", r.job.ID)
 
+	scheduledAt := time.Now().Add(time.Duration(-1*rand.Intn(6400)) * time.Millisecond)
```

Comment:
this is just a placeholder while we wait for the API, and is only here for demonstration purposes